tests/Language/PureScript/Ide/CompletionSpec.hs

```diff
@@ -21,6 +24,33 @@ reexportMatches =
 matches :: [(Match IdeDeclarationAnn, [P.ModuleName])]
 matches = map (\d -> (Match (mn "Main", d), [mn "Main"])) [ ideKind "Kind", ideType "Type" Nothing [] ]
 
+
+-- | Type
+-- { typeSearch        :: Text
+-- , typeFilters       :: [Filter]
+-- , typeCurrentModule :: Maybe P.ModuleName
+-- }
+
+typ :: Text -> Command
+typ txt = Type txt [] Nothing
+
+load :: [Text] -> Command
+load = LoadSync . map Test.mn
+
+rebuildSync :: FilePath -> Command
+rebuildSync fp = RebuildSync ("src" </> fp) Nothing
+
+-- spec :: Spec
+-- spec = describe "Rebuilding single modules" $ do
+--     it "rebuilds a correct module without dependencies successfully" $ do
```

Comment:
I don't really mind the Type command comment, but this one should be deleted.

Reply:
ugh I guess I was tired, such half-assery. thanks